test_runner/regress/test_sharding.py

```diff
@@ -1243,3 +1244,94 @@ def test_sharding_unlogged_relation(neon_env_builder: NeonEnvBuilder):
 
         # Ensure that post-endpoint-restart modifications are ingested happily by pageserver
         wait_for_last_flush_lsn(env, ep, tenant_id, timeline_id)
+
+
+# Stripe sizes in number of pages.
+TINY_STRIPES = 16
+LARGE_STRIPES = 32768
+
+
+@pytest.mark.parametrize("stripe_size", [TINY_STRIPES, LARGE_STRIPES])
+def test_sharding_compaction(neon_env_builder: NeonEnvBuilder, stripe_size: int):
```

Comment:
nit: Might be a good fit for `test_compaction.py`

Reply:
Did you intend to keep the test in `test_sharding` too?

Reply:
Oops, no.  Removed.